### DISCUSSION TITLE
articles/notes-of-titans

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#132] [Comments]
https://hutusi.com/articles/notes-of-titans 

 知识管理达人采铜推荐了《巨人的工具》，我在微信读书上找来此书看完。作者蒂姆·费里斯是一名播客主播，在他的播客中他采访了美国各业界精英，整理成了《巨人的工具》一书。各位名流根据自己的经验给出一些关于健康、财富和人生智慧方面的建议。说实话，我觉得这本书可以当成闲暇的读物，按照自己的兴趣读几章还是可以的，我就是这么快速看完的，特别是第一部分的健康篇几乎没看，因为我觉得所谓健康方面的建议还是要根据个人实际情况来定，不要盲目仿效为好。不过有些人物的访谈也挺有意思，给我印象最深刻的是对凯文·凯利和纳瓦尔·拉维康特的采访。

**Comment by xibolun** [comment]
感谢老哥推荐，加入todo reader

**Comment by hutusi** [comment]
> @xibolun
> 感谢老哥推荐，加入todo reader

谢谢评论，很高兴有人一起读。

**Comment by sunyunxian** [comment]
同读过，对里面关于一些冥想、journal、morningpage 的概念非常认同。

**Comment by hutusi** [comment]
> 同读过，对里面关于一些冥想、journal、morningpage 的概念非常认同。

我是坚持在坚持日记，不过冥想还没能坚持下来~~
